android/app/src/main/java/org/fossasia/openevent/activities/auth/ChangePasswordActivity.java

```diff
@@ -77,6 +68,12 @@ protected void onCreate(Bundle savedInstanceState) {
         });
     }
 
+    private void getUser() {
+        changePasswordActivityViewModel.getUser().observe(this, user -> {
+            this.user = user;
+        });
+    }
+
     private void saveChanges() {
         if (currentPasswordInput == null || newPasswordInput == null || confirmPasswordInput == null)
```

Comment:
These can't be null I guess, please confirm by removing the checks. If the views are null. I think the app **should** crash for good and not silently keep working